README.md

```diff
@@ -86,21 +99,27 @@ Copy and configure:
 ```.env.dist``` to  ```.env```
 ```config/packages/parameters.yml.dist``` to ```config/packages/parameters.yml```
 
-### 3. Build public assets
+### 4. Create env local file
 
 ```
  composer dump-env prod
- yarn encore prod 
 ```
 
-### 4. Create database and schema 
+### 5. Build public assets
+
+```
+ yarn encore prod
+ ./bin/console assets:install
+```
+
+### 6. Create database and schema 
```

Comment:
This seems to duplicate paragraph 4, but not fully. I think you should merge them